src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let menFiltered = people.filter(i => i.sex === 'm');
+
+  menFiltered = century
+    ? menFiltered.filter(person => Math.ceil(person.died / 100) === century)
+    : menFiltered;
```

Comment:
I think in this case it's better to use if (more readable)

```suggestion
  if (century) {
    menFiltered = menFiltered
    .filter(person => Math.ceil(person.died / 100) === century)
  }
```